frontend/public/locales/en/common.json

```diff
@@ -99,6 +99,20 @@
   "apps-by-developer": "Apps by {{developer}}",
   "apps-by-projectgroup": "Apps in the {{projectgroup}} group",
   "search-for-query": "Search for '{{query}}'",
+  "apps-description": "Explore the Linux apps available on Flathub",
```

Comment:
Are you using this and the next line? Or is this from a conflict resolved wrong?